pom.xml

```diff
@@ -25,30 +25,37 @@
 		<developer>
 			<id>aaron</id>
 			<name>Aaron Petkau</name>
-			<email>aaron.petkau@phac-aspc.gc.ca</email>
+			<email>aaron.petkau@canada.ca</email>
 			<organization>Public Health Agency of Canada</organization>
-			<organizationUrl>http://www.phac-aspc.gc.ca/</organizationUrl>
+			<organizationUrl>http://www.canada.ca/</organizationUrl>
```

Comment:
https://www.canada.ca/en/public-health.html

Reply:
Create idea, updated in 73107a3eefe0cecb2ef4e53fadb3369e63716095